jobs/secureproxy/spec

```diff
@@ -19,3 +19,21 @@ properties:
   secureproxy.custom_server_config:
     description: Custom nginx configuration to include in main http block
     default: ""
+
+  secureproxy.tic.whitelist:
+    description: Hash mapping email domains to lists of whitelisted CIDRs
+    example:
+      foo.gov:
+      - 192.0.2.0/24
+      - 198.51.100.101/32
+    default: {}
+  secureproxy.tic.host_whitelist:
+    description: List of hostnames to include in TIC filter
+    example:
+    - api.fr.cloud.gov
+    default: []
+  secureproxy.tic.path_whitelist:
+    description: List of unauthenticated paths to exempt from TIC filter
+    example:
+    - ^/v2/info
```

Comment:
I think this should be tied to the hostname it's for.  Maybe the `host_whitelist` should contain a list of paths to exclude?  Something like:
```yaml
 secureproxy.tic.host_whitelist:
  - hostname: api.fr.cloud.gov
    exclude:
    - /v2/info
```

What do you think?